randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1341,8 +1339,20 @@ Extra - room_id: [35]
   * Layers: default
   * Tunnel to Diffusion Connection Access/Other to Powamp Shaft
   * Extra - door_idx: (112,)
+  > Top of Ladder
+      Missiles
+
+> Top of Ladder; Heals? False
```

Comment:
I havent looked at this from in-editor yet only this review, but i feel like either this node should be renamed to "top tunnel" or something, or the other connections are missing a few requirements.

Reply:
Changed to "In Narrow Tunnel" since we're way above the ladder